app/views/other-resources.html

```diff
@@ -11,14 +11,16 @@
           </div>
           <alerts alerts="alerts"></alerts>
           <div class="data-toolbar">
-            <ui-select ng-model="kindSelector.selected" theme="bootstrap" search-enabled="true" ng-disabled="kindSelector.disabled" title="Choose a resource" style="width: 250px;">
-              <ui-select-match placeholder="Choose a resource to list...">{{$select.selected.kind | humanizeKind : true}}</ui-select-match>
-              <ui-select-choices repeat="kind in kinds | filter : {kind: $select.search} : matchKind | orderBy : 'kind'">
-                <div ng-bind-html="(kind.kind | humanizeKind : true) | highlight: $select.search"></div>
-              </ui-select-choices>
-            </ui-select>
+            <div class="col-xs-4 col-sm-4 col-md-2 pad-left-none pad-right-none">
```

Comment:
How does this look at mobile screen widths (~350px or so)? `col-xs-4` sounds really small.

Typically the `col-*` classes go inside a `<div class="row">`, and you would want to use them for other items in the toolbar like the filter. I think instead of `col-*`, I'd just create a new class that sets the widths and padding for this control.